modules.py

```diff
@@ -247,7 +247,10 @@ def multihead_attention(queries,
         
         # Restore shape
         outputs = tf.concat(tf.split(outputs, num_heads, axis=0), axis=2 ) # (N, T_q, C)
-              
+        
+        # Linear projections
+        outputs = tf.layers.dense(outputs, num_units, activation=tf.nn.relu) # (N, T_q, C)
```

Comment:
I think you are right about an extra projection, but can I ask about the activation function here? seems in the original paper there is no bias and activation, only a plain  "Concat(head1, ..., headh)W_O"